src/App.js

```diff
@@ -3,6 +3,7 @@ import Home from './components/Home';
 import Form from './components/Form';
 import Error from './components/Error';
 import NavBar from './components/NavBar';
+import FormError from './components/FormError';
```

Comment:
Don't need this App.js because it is never used only need it to imported in components that use it